spec/proto/extension/v1/oss.proto

```diff
@@ -0,0 +1,1059 @@
+//The file defined base on s3 protocol, to get an in-depth walkthrough of this file, see:
+//https://docs.aws.amazon.com/s3/index.html
+//https://github.com/aws/aws-sdk-go-v2
+syntax = "proto3";
+
+package spec.proto.extension.v1;
+
+import "google/protobuf/empty.proto";
+
+option go_package = "mosn.io/layotto/spec/proto/extension/v1;s3";
+
+service ObjectStorageService{
+    //Init oss client
+    rpc InitClient(InitInput) returns (google.protobuf.Empty){}
+    //Adds an object to a bucket.
+    //Refer https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObject.html
+    rpc PutObject(stream PutObjectInput) returns(PutObjectOutput){}
+    //Retrieves objects.
+    //Refer https://docs.aws.amazon.com/AmazonS3/latest/API/API_GetObject.html
+    rpc GetObject(GetObjectInput) returns (stream GetObjectOutput){}
+    //Delete objects.
+    //Refer https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObject.html
+    rpc DeleteObject(DeleteObjectInput) returns (DeleteObjectOutput){}
+    //Sets the supplied tag-set to an object that already exists in a bucket.
+    //Refer https://docs.aws.amazon.com/AmazonS3/latest/API/API_PutObjectTagging.html
+    rpc PutObjectTagging(PutObjectTaggingInput) returns (PutObjectTaggingOutput){}
+    //Removes the entire tag set from the specified object.
+    //Refer https://docs.aws.amazon.com/AmazonS3/latest/API/API_DeleteObjectTagging.html
+    rpc DeleteObjectTagging(DeleteObjectTaggingInput) returns(DeleteObjectTaggingOutput){}
+    //Returns the tag-set of an object.
+    //Refer https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_GetObjectTagging.html
+    rpc GetObjectTagging(GetObjectTaggingInput) returns(GetObjectTaggingOutput){}
+    //Creates a copy of an object that is already stored in oss server.
+    //Refer https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_CopyObject.html
+    rpc CopyObject(CopyObjectInput) returns(CopyObjectOutput){}
+    //Delete multiple objects from a bucket.
+    //Refer https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_DeleteObjects.html
+    rpc DeleteObjects(DeleteObjectsInput) returns(DeleteObjectsOutput){}
+    //Returns some or all (up to 1,000) of the objects in a bucket.
+    //Refer https://docs.aws.amazon.com/zh_cn/AmazonS3/latest/API/API_ListObjects.html
+    rpc ListObjects(ListObjectsInput) returns(ListObjectsOutput){}
+    //Returns object canned acl.
+    //Refer https://docs.aws.amazon.com/AmazonS3/latest/userguide/acl-overview.html#CannedACL
+    rpc GetObjectCannedAcl(GetObjectCannedAclInput) returns(GetObjectCannedAclOutput){}
```

Comment:
Coud u please provide the documentation URL of `GetObjectCannedAcl` and `PutObjectCannedAcl `?
The url you put here only explains what is `cannedACL`. 
But I can't find the `get` and `put` API in the aws s3 doc https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/s3#Client.GetObjectAcl

Reply:
> ![image](https://user-images.githubusercontent.com/26001097/179241206-aad2d510-b6ef-426a-be0d-79fecead4a87.png) The design doc is a binary file , not a markdown file written with text. Please fix it

done